### ISSUE TITLE
Exception thrown parsing user-agent headers for iOS

### ISSUE THREAD

**Issue by timmc-edx** [jotes/django-cookies-samesite#28]
* django-cookies-samesite version: 0.6.1
* Django version: 2.2
* Python version: 3.5
* Operating System: Ubuntu

I'm seeing a stack trace on a large number of requests, I think mostly or entirely mobile users. Here's a reproduction case:

```
>>> django_cookies_samesite.user_agent_checker.UserAgentChecker("edX/2.22.0 CFNetwork/1121.2.2 Darwin/19.3.0").do_not_send_same_site_policy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/timmc/openedx/edx-platform/venv-3.5/lib/python3.5/site-packages/django_cookies_samesite/user_agent_checker.py", line 32, in do_not_send_same_site_policy
    return not (self.supported_browsers_os() or self.other_browsers())
  File "/home/timmc/openedx/edx-platform/venv-3.5/lib/python3.5/site-packages/django_cookies_samesite/user_agent_checker.py", line 35, in supported_browsers_os
    return self.supported_ios_and_mac_os_browsers() or self.supported_chrome_and_uc_browsers()
  File "/home/timmc/openedx/edx-platform/venv-3.5/lib/python3.5/site-packages/django_cookies_samesite/user_agent_checker.py", line 41, in supported_ios_and_mac_os_browsers
    return self.is_supported_ios_version() or self.is_supported_mac_osx_safari()
  File "/home/timmc/openedx/edx-platform/venv-3.5/lib/python3.5/site-packages/django_cookies_samesite/user_agent_checker.py", line 80, in is_supported_ios_version
    return self.is_ios() and not (int(self.user_agent_os.get('major', '0')) == self.MIN_IOS_VERSION)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

**Comment by timmc-edx** [comment]
Looks like this line is the culprit: https://github.com/jotes/django-cookies-samesite/pull/21/files#diff-f1123f677dd9c53ef31c510f990d4e7dR80

```python
>>> c.user_agent_os
{'patch': None, 'patch_minor': None, 'family': 'iOS', 'minor': None, 'major': None}
```

**Comment by jotes** [comment]
@timmc-edx I released `0.6.2` with the fix. Can you confirm that it solves the issue for you?

**Comment by timmc-edx** [comment]
Sorry for the very late reply, but yes, this seems to have fixed it. :-) Thank you!
